application/gqlgen/schema.graphql

```diff
@@ -207,11 +208,15 @@ input UpdateOrganizationInput {
 type OrganizationDomain {
     domain: String!
     organizationID: ID!
+    authType: String!
+    authConfig: String!
 }
 
 input CreateOrganizationDomainInput {
```

Comment:
Should we have a separate UpdateOrganizationDomainInput that allows the user to modify the domain? I guess we would need two domain fields - one to find the current object and a second for the new domain value.

Reply:
i don't think we need to allow them to update the domain name itself, they can just recreate it.